lodash.src.js

```diff
@@ -7325,25 +7325,17 @@
      * _.map(_.sortByAll(users, ['user', 'age']), _.values);
      * // => [['barney', 26], ['barney', 36], ['fred', 30], ['fred', 40]]
      */
-    function sortByAll() {
```

Comment:
> because of it's inclusion in ES6 but I'm glad regardless

I'm not likely to take any more suggestions from you or Justin if the treatment I get is you all rolling over when presented to the Underscore crowd. Now you leave me in a shitty position of having to take their crap name or potentially causing compat issues if they work it into Backbone (which I believe is the end goal). All while Adam did his uninformed and inaccurate bemoaning of lodash in the Underscore issue that I'm unable to respond to.